platform/linux-generic/odp_comp.c

```diff
@@ -133,7 +133,7 @@ static void process_input(odp_packet_t pkt_out,
 	do {
 		out_data =
 		    odp_packet_offset(pkt_out, start, &out_len, &cur_seg);
-		ODP_DBG("out_data 0x%x seg_data_ptr 0x%x out_len %d seg 0x%x\n",
+		ODP_DBG("out_data %p seg_data_ptr %p out_len %d seg %p\n",
```

Comment:
I wonder if we want to use %p format for handles. Of course, with the format warning enaled, we will get a warning from this format if segment handle changes to a non-pointer. Or maybe this should not even try to print the handle directly, but use odp_packet_seg_to_u64()?

Reply:
We should probably just use to_u64() everywhere for all handles. I didn't want to do this for the handles that happen to be visible in this commit, but I could try to go through them systematically in a separate commit.

Reply:
I am ok with not changing to use to_u64() in this commit. We anyway get the warning if the format gets out of sync with the handle type.